### DISCUSSION TITLE
A way to shuffle windows of the same app

### DISCUSSION THREAD

**Discussion by alzaki-clgx** [rxhanson/RectanglePro-Community#288] [Ideas]
hi Happy user of Rectangle Pro here
I have a App Layout which has two entries for Zoom
However, it puts Zoom Chat in the left footprint and Zoom Meeting in the right footprint
Is there a way to record a Keyboard Shortcut to rotate these Zoom windows?

Oh! How about this, if you execute the shortcut for the App Layout again, then it reinforces the layout but with a different rotation for windows of the same app

**Comment by rxhanson** [comment]
Thanks for the feedback. You can click on the + button when you hover on a row in an app layout to add a repetition for that app window, where repeating the shortcut will cycle through your configured repetitions. Maybe that’s the best solution here? At least maybe the best existing solution.

**Comment by alzaki-clgx** [comment]
Success! I understand that I was able to add a _Repetition_ row underneath my Zoom row. But what does adding another _app_ row underneath my Zoom row mean? I am just now learning that App layouts aren't just singular lists of app dimensions but can also contain sublists
